docs/framework/wpf/advanced/opentype-font-features.md

```diff
@@ -58,7 +58,7 @@ This topic provides an overview of some of the key features of [!INCLUDE[TLA#tla
   
  The following text displays superscripts for the Palatino Linotype font.  
   
- ![Text using OpenType superscripts](../../../../docs/framework/wpf/advanced/media/opentypefont14.gif "opentypefont14")  
+ ![Text using OpenType superscripts](../../../../docs/framework/wpf/advanced/media/opentype-font-features/opentype-superscripts.gif "Text using OpenType superscripts")  
 Text using OpenType superscripts  
```

Comment:
Ok, 1st topic and we already run into unusual things. 😄 
This topic has a label under the image. So I'd remove this label from under the images, otherwise, screen readers will read this twice.
But I don't think this will be very common.